app/grandchallenge/core/management/commands/transfer_notifications.py

```diff
@@ -0,0 +1,201 @@
+                notification.save()
+                num_notifications += 1
+            elif notification.action.actor_content_type.model == "submission":
+                notification.type = (
+                    NotificationType.NotificationTypeChoices.MISSING_METHOD
+                )
+                notification.actor_content_type = ContentType.objects.filter(
+                    model="user"
+                ).get()
+                notification.actor_object_id = (
+                    notification.action.actor.creator.pk
+                )
+                notification.action_object_content_type = (
+                    notification.action.actor_content_type
+                )
+                notification.action_object_object_id = (
+                    notification.action.actor_object_id
+                )
+                notification.target_content_type = (
+                    notification.action.target_content_type
+                )
+                notification.target_object_id = (
+                    notification.action.target_object_id
+                )
+                notification.verb = notification.action.verb
+                notification.save()
+                num_notifications += 1
+            elif notification.action.actor_content_type.model == "algorithm":
+                notification.type = (
+                    NotificationType.NotificationTypeChoices.JOB_STATUS
+                )
+                notification.actor_content_type = (
+                    notification.action.target_content_type
+                )
+                notification.actor_object_id = (
+                    notification.action.target_object_id
+                )
+                notification.target_content_type = (
+                    notification.action.actor_content_type
+                )
+                notification.target_object_id = (
+                    notification.action.actor_object_id
+                )
+                notification.description = notification.action.description
+                notification.verb = notification.action.verb
+                notification.save()
+                num_notifications += 1
+            elif (
+                notification.action.actor_content_type.model
+                == "rawimageuploadsession"
+            ):
+                notification.type = (
+                    NotificationType.NotificationTypeChoices.IMAGE_IMPORT_STATUS
+                )
+                notification.action_object_content_type = (
+                    notification.action.actor_content_type
+                )
+                notification.action_object_object_id = (
+                    notification.action.actor_object_id
+                )
+                notification.verb = notification.action.verb
+                notification.save()
+                num_notifications += 1
+        self.stdout.write(
+            self.style.SUCCESS(f"{num_notifications} notifications processed")

```

Comment:
Maybe it would be nice to ad the count for all `Notification` objects here. That way we can spot if any notifications were somehow missed in the conversion step.